pages/landing/hipaa/template/pricing/index.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Pricing
+excerpt: Configure the plan just right for your team.
+permalink: /hipaa-cloud-compliance/pricing/
+
+pricing:
```

Comment:
Hm, from this pricing page, a few things are not clear:

1. What _is_ the product? By the time I get to the pricing page, it isn't obvious that I get IaC, or a fully deployed Ref Arch, or a consulting engagement, or what? How am I supposed to use the product? What do I still have to do after I get it?
1. What are you paying for now? Is HIPAA fully available or are we still working on it?